composer.lock

```diff
@@ -4,37 +4,40 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "d2972469edcc7b78f4cc9f3d64ca9323",
```

Comment:
I see you did `composer update`. Are we wanting massive changes to the dependencies for this fork? Or just a single change that you did?